client/src/Components/Common/DropList/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Select } from 'antd';
+import PropTypes from 'prop-types';
+
+const { Option } = Select;
+
+const DropList = ({ options, onChange }) => (
+  <Select
+    onChange={onChange}
+    labelInValue
+    defaultValue={{ value: 1 }}
+    style={{ width: 130 }}
```

Comment:
There's no need to put the width here and if it's necessary you can pass the width as a prop and the default value should be passed as a prop and you should pass the placeholder to the component and use it.